docs/units-and-global-variables.rst

```diff
@@ -160,6 +160,8 @@ more details on error handling and when to use which function.
 
 .. index:: keccak256, ripemd160, sha256, ecrecover, addmod, mulmod, cryptography,
 
+.. _mathematical-and-cryptographic-functions:
```

Comment:
Is this used anywhere now?

Reply:
Yes, this is actually a fix for the bug that triggered this whole cleanup. `micropayment.rst` was referring to this ref but it did not exist. I think this used to have a page of its own but was later incorporated into `units-and-global-variables.rst` and breakage was not spotted due to it not using the `` :ref:`xyz` `` syntax.